examples/basic/.gitignore

```diff
@@ -0,0 +1 @@
+docs
```

Comment:
Running typedoc on the examples folder locally, I needed to gitignore this folder.